### PR TITLE
Updating sphinx to py3

### PR DESCRIPTION
**Pull request by Eldinnie**

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -291,17 +291,9 @@
 
 # -- script stuff --------------------------------------------------------
 
-import inspect
-
 
 def autodoc_skip_member(app, what, name, obj, skip, options):
```
Comment (bomjacob) [inline docs/source/conf.py]: We can probably remove the whole "script stuff" and below now :)

**Comment by tsnoam** [comment]
@Eldinnie @bomjacob 
I think this is good merge, no?

File: docs/source/conf.py (hunk 1 above)
Comment (Eldinnie) [reply docs/source/conf.py]: It's a part of the template so leaving it in.

**Comment by Eldinnie** [comment]
Yes

**Pull request merged**